### ISSUE TITLE
Feature request: libhandy support

### ISSUE THREAD

**Issue by FedericoCeratto** [StefanSalewski/gintro#89]
Hello and thanks for developing gintro.
Libhandy[1] is gaining popularity on phones (Librem, PinePhone) and tablets. Would you consider adding wrappers/helpers for libhandy as an optional dependency?

[1] https://gitlab.gnome.org/GNOME/libhandy/ https://gnome.pages.gitlab.gnome.org/libhandy/ https://repology.org/project/libhandy/versions

Thanks

**Comment by StefanSalewski** [comment]
Yes, I also considered adding it. Unfortunately I have currently no mobile device with GTK support, so I may not be able to test it at all. How soon do you need it?

**Comment by FedericoCeratto** [comment]
Not urgent. Also, libhandy is desiged to generate UIs that work well on small screens but also on desktops, e.g. https://gnome.pages.gitlab.gnome.org/libhandy/visual-index.html - so it can be tested on any desktop :)

**Comment by StefanSalewski** [comment]
OK, I have added libhandy. Was basically only one more line in the generator script.

Note that libhandy is not yet available for GTK4.

I tested with the latest libhandy version from

https://gitlab.gnome.org/GNOME/libhandy/

much older versions may not work yet. I converted the example.py Python example to Nim, it is available in examples/gtk3/handy.nim

Currently I can not provide more examples or tutorials.

Let us know when there are issues.
